tests/rflx-arrays-tests.adb

```diff
@@ -25,7 +25,13 @@ package body RFLX.Arrays.Tests is
    Data : RFLX_Builtin_Types.Bytes (RFLX_Builtin_Types.Index'First .. RFLX_Builtin_Types.Index'First + 1) :=
      (others => 0);
 
-   procedure Write_Data (Buffer : out RFLX_Builtin_Types.Bytes) is
+   function Data_Length (L : RFLX_Builtin_Types.Length) return Boolean is
```

Comment:
`Data_Length` is not a very descriptive name. (or I misinterpret what it's doing)

Reply:
I also think that we should look for a better name. We should also change it in the README then. Some ideas: `Valid_Data_Length`, `In_Data_Range`.

Reply:
I think `Valid_Data_Length` is a good choice.